br/cmd/br/fips.go

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 //go:build boringcrypto
-// +build boringcrypto
```

Comment:
https://github.com/pingcap/tidb/blob/8c3210b9493c54a8221b6632fb9fad006ba46589/Makefile.common#L32-L34

Used here... can you verify this with your change?  

```
ENABLE_FIPS=1 make
```

Reply:
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

Reply:
./bin/tidb-server -V                                             ✔  1m 39s  17:10:31 
Release Version: v8.0.0-alpha-151-g975a8fba18-dirty
Edition: Community
Git Commit Hash: 975a8fba18cc282bbefcc223db9cb1e57feafe67
Git Branch: add_lint_check
UTC Build Time: 2024-01-30 09:08:52
GoVersion: go1.21.0 X:boringcrypto
Race Enabled: false
Check Table Before Drop: false
Store: unistore